### ISSUE TITLE
Site review by Ashley

### ISSUE THREAD

**Issue by tansaku** [AgileVentures/AgileVentures#10]
very interesting AV review document from Ashley

it highlights how users coming to home page may be confused about what we do ...

[agileventures_experience_review.pdf](https://github.com/AgileVentures/AgileVentures/files/550378/agileventures_experience_review.pdf)

**Comment by weedySeaDragon** [comment]
(btw: here on github, this is me: _weedySeaDragon_  )

**Comment by weedySeaDragon** [comment]
I'm going to add another pain point:  Soooo many groups in Slack!  (How to find the signals in  the noise?)

#### Pain Point
As someone new to AV
I want to participate in discussions and conversations in Slack
In order to be active in the project(s) I'm involved in 
And to learn from other AV members
And to connect with other AV members

But since Slack allows anyone and everyone to create a channel, it's hard to know that there are channels that would be helpful to me and of interest to me.

It's clear that the channel associated with a project.  But beyond that, it's hard to know what other channels might be helpful.

#### Suggestions

- Have a place (page?) on AV that lists the 'main' channels

-  Generate a list of channels that are for projects that are no longer active and provide some *signpost/clue*  (via renaming the channel?) that they are stale.

  - Is there a way to get info from Slack to get a list of stale channels? (channels that haven't had activity for a long time)  ex: via API?

- Guideline for naming a new Slack channel so that glancing at the name of a channel immediately provides information about the purpose of the channel.  
  - It may be too hard to enforce/audit the names of new channels since there are so many new people that join AV.  Instead of trying to enforce naming conventions for new channels created by most folks, just enforce naming conventions for channels that are for and about AV.

**Comment by tansaku** [comment]
good point - we can link in slack channels into every project description ... ideally we'd use the slack API so you could select your slack channel from a list in the project creation portion ... 

![](https://www.dropbox.com/s/790ml7rpg79yzio/Screenshot%202016-10-31%2017.20.37.png?dl=1)

**Comment by tansaku** [comment]
annoyingly the description doesn't hyperlink - we have an outstanding ticket for that: https://github.com/AgileVentures/WebsiteOne/issues/769 - we could get that done and then go round and edit all the descriptions to follow the above, while creating a ticket to specifically associate the slack channel - connecting it to make sure it's valid
